File: 2st/Riyeon/코드 처리하기.java
class Solution {
    public String solution(String code) {
        String answer = "";
        int mode = 0;
        for(int i=0; i<code.length(); i++){
            char ch = code.charAt(i);
            if(ch == '1') mode = mode==0? 1: 0;
            else {
                if(mode==1 && i%2==1) answer += ch;
                else if(mode==0 && i%2==0) answer += ch;
            }
        }
        if(answer.isEmpty()) answer = "EMPTY";
        return answer;
    }
}

File: 2st/Riyeon/코드 처리하기.py
def solution(code):
    mode = 0
    ret = ''
    for i in range(len(code)):
        if mode == 0:
            if code[i] == '1':
                mode = 1
            elif i % 2 == 0:
                ret += code[i]
        else:
            if code[i] == '1':
                mode = 0
            elif i % 2 == 1:
                ret += code[i]            
    return ret if len(ret) != 0 else 'EMPTY'